.readthedocs.yml

```
@@ -3,19 +3,20 @@
 
 version: 2
 build:
-  os: ubuntu-22.04
+  os: ubuntu-24.04
   tools:
     python: "3"
   jobs:
+    create_environment:
+      - asdf plugin add uv
+      - asdf install uv latest
+      - asdf global uv latest
+    install:
+      - UV_PROJECT_ENVIRONMENT=$READTHEDOCS_VIRTUALENV_PATH uv sync --frozen --group docs
     pre_build:
-      - make -C doc apidoc
-      - make -C doc relnotes
+      - uv run make -C doc apidoc
+      - uv run make -C doc relnotes
```

Maybe this is needed to make sure no extra venvs are created?
```suggestion
      - UV_PROJECT_ENVIRONMENT=$READTHEDOCS_VIRTUALENV_PATH uv run make -C doc apidoc
      - UV_PROJECT_ENVIRONMENT=$READTHEDOCS_VIRTUALENV_PATH uv run make -C doc relnotes
```

---

This change has been added